src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  id: number;
+  userId: number;
```

Comment:
Consider removing the `userId` property from the `Todo` and `NewTodo` interfaces if it's not needed for this specific task. Otherwise, the code looks good. 🚀